extensions/stackdriver/stackdriver.cc

```diff
@@ -346,8 +344,7 @@ inline bool StackdriverRootContext::enableEdgeReporting() {
 
 bool StackdriverRootContext::shouldLogThisRequest() {
   std::string shouldLog = "";
-  if (!getValue({"filter_state", ::Wasm::Common::kAccessLogPolicyKey},
-                &shouldLog)) {
+  if (!getValue({::Wasm::Common::kAccessLogPolicyKey}, &shouldLog)) {
```

Comment:
I don't see this was declared as property anywhere. We should have test for this, wondering why test does not fail.

Reply:
It's set by access log policy extension. Declaration is not necessary, it'll default to a default declaration https://github.com/envoyproxy/envoy-wasm/commit/80e1a645c25f6243bfacf0eb5b9e96eb2e767aff#diff-965399965695ac238aae83e5d9bd5666R42. Get/set/declare now all use the same identifier for consistency.

Reply:
I see. `property` declaration is only needed for CEL expression. Find a typo during code reading: https://github.com/envoyproxy/envoy-wasm/blob/f80bba648062e95b98497b84ba20a8165828b082/source/extensions/common/wasm/context.cc#L428

Reply:
@jplevyak might want to fix the macro pollution ^^^

Reply:
getProperty should also be able to read inside buffers without using CEL compiler.